### PR TITLE
Add Level class

### PR DESCRIPTION
**Pull request by lokidoki102**
Add Level class to support add house and seller command.

Add test cases to test for the validity of level inputs such as null input and invalid format.

Add a prefix for level to support CLI commands.

### CODE REVIEW

File: src/main/java/seedu/address/model/house/Level.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        requireNonNull(level);
+        checkArgument(isValidLevel(level), MESSAGE_CONSTRAINTS);
+        value = level;
+    }
+
+    /**
+     * Returns true if a given string is a valid level.
+     */
+    public static boolean isValidLevel(String test) {
+        return test.matches(VALIDATION_REGEX);
```
Comment (KhoonSun47) [inline src/main/java/seedu/address/model/house/Level.java:33]: Looks good, but I think I will give the same comment as the "Block" class. Is it possible that users can key in '0' or '00' for level? I am not too sure whether there is 0 level for HDB/Condo. However, if you done your research and there is 0 level, then ignore my comment on this.

**Review by KhoonSun47 — APPROVED** [review]
LGTM! Maybe take note of my comments. Thanks :)

**Review by felixchanyy — APPROVED** [review]
LGTM

**Review by felixchanyy — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 2]
```diff
@@ -137,5 +139,31 @@ public static Level parseLevel(String level) throws ParseException {
             throw new ParseException(Level.MESSAGE_CONSTRAINTS);
         }
         return new Level(trimmedLevel);
+
```
Comment (KhoonSun47) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:142]: missing `/**` after merge

**Review by KhoonSun47 — DISMISSED** [review]

**Review by zengzihui — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 3]
```diff
@@ -11,6 +11,7 @@ public class CliSyntax {
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_LEVEL = new Prefix("level/");
```
Comment (felixchanyy) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java:14]: Perhaps we could use 'l/' instead of 'level/'?
